doc/README.md

```diff
@@ -0,0 +1,421 @@
+# Seedvault App Backup Repository Documentation
+
+This document is a technical description of how Seedvault backs up apps
+installed on an Android device.
+
+It is inspired by the design of the [Restic backup program](https://restic.net/).
+Portions of this document are blatant copies of
+[its documentation](https://restic.readthedocs.io/en/latest/100_references.html).
+However, due to the different nature of Android backups,
+the design [was heavily simplified](#differences-to-restic).
+
+Note that the backup of files is described [in a different document](../storage/doc/design.md).
+
+## Terminology
+
+This section introduces terminology used in this document.
+
+*Repository*: All data produced during a backup is sent to and stored in a repository
+in a structured form.
+
+*Chunk*: Larger files are cut into re-usable chunks that are the unit of de-duplication.
+
+*Blob*: A blob is a chunk stored encrypted and compressed in the repository.
+
+*Snapshot*: A snapshot stands for the state of a collection of apps
```

Comment:
[Clarity] If a blob is a file, and a snapshot is a file, then the definition of a snapshot ought to similarly emphasize that it is encrypted.

Reply:
Thanks, I used your suggestion.